apps/condo/jest.config.js

```diff
@@ -3,14 +3,23 @@ module.exports = {
         {
             displayName: 'schema',
             testEnvironment: 'node',
-            testMatch: [`${__dirname}/schema/**/*.test.js`, `${__dirname}/domains/**/schema/*.test.js`],
+            testMatch: [
+                `${__dirname}/schema/**/*.test.js`,
```

Comment:
It’s inside the common. Why we need it?

Reply:
I've just reformatted code....Nothing was added 😄